torch/csrc/distributed/rpc/request_callback_impl.cpp

```diff
@@ -129,9 +121,48 @@ std::shared_ptr<FutureMessage> RequestCallbackImpl::processRpc(
         // addForkOfOwner as it is not a fork. To allow callee to distinguish
         // when this request is sent to self, the caller will set forkId using
         // rrefId (OwnerRRef does not have a forkId anyway).
-        ctx.addForkOfOwner(rrefId, forkId);
+        RRefContext::getInstance().addForkOfOwner(rrefId, forkId);
```

Comment:
What is the reason for this change?

Reply:
I just reverted this aspect of the change. It's a vestige of an intermediate version, where I was placing this in a callback and wanted to minimize captures.